tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Utils/EmbeddedResourceReader.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Security.Cryptography.X509Certificates;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Utils
+{
+    public class EmbeddedResourceReader
+    {
+        private const string EmbeddedResourceQualifier = "NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests";
+
+        public static X509Certificate2 GetCertificate()
+        {
+            var resourceName = $"{EmbeddedResourceQualifier}.public_privatekey.pfx";
+            using (var certificateStream =
+                typeof(EmbeddedResourceReader).Assembly.GetManifestResourceStream(resourceName))
+            {
+                if (certificateStream == null)
+                {
+                    return null;
+                }
+
+                var rawBytes = new byte[certificateStream.Length];
+                for (var i = 0; i < certificateStream.Length; i++)
+                {
+                    rawBytes[i] = (byte)certificateStream.ReadByte();
+                }
+
+                return new X509Certificate2(rawBytes, "12345", X509KeyStorageFlags.UserKeySet);
+            }
```

Comment:
Could this perhaps be rewritten using the C# 8 syntax?

instead of:
```
            using (var certificateStream =
                typeof(EmbeddedResourceReader).Assembly.GetManifestResourceStream(resourceName))
            {
                if (certificateStream == null)
                {
                    return null;
                }

                var rawBytes = new byte[certificateStream.Length];
                for (var i = 0; i < certificateStream.Length; i++)
                {
                    rawBytes[i] = (byte)certificateStream.ReadByte();
                }

                return new X509Certificate2(rawBytes, "12345", X509KeyStorageFlags.UserKeySet);
            }
```
like this:
```
            using var certificateStream = typeof(EmbeddedResourceReader).Assembly.GetManifestResourceStream(resourceName);

            if (certificateStream == null)
            {
                return null;
            }

            var rawBytes = new byte[certificateStream.Length];
            for (var i = 0; i < certificateStream.Length; i++)
            {
                rawBytes[i] = (byte)certificateStream.ReadByte();
            }

            return new X509Certificate2(rawBytes, "12345", X509KeyStorageFlags.UserKeySet);

```